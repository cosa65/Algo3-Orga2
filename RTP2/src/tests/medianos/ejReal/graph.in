2 0.85 1 graph.out 0.0001
