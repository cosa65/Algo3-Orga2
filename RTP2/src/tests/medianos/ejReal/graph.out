# Directed graph (each unordered pair of nodes is saved once):
# Example generated by webparser.py
# Nodes: 14 Edges: 20 
# FromNodeId   ToNodeId
2	8
2	9
2	11
3	11
3	9
3	2
3	8
4	6
4	7
4	12
6	4
7	6
7	4
8	11
8	2
8	9
9	8
9	2
11	2
11	8
